src/klein/test/test_headers_compat.py

```diff
@@ -1,4 +1,4 @@
-# -*- test-case-name: klein.test.test_headers -*-
+# -*- test-case-name: klein.test.test_headers_compat -*-
```

Comment:
we should probably have a lint test for the name here matching.

Reply:
+1